src/Tizen.NUI.Components/Controls/TabContent.cs

```diff
@@ -0,0 +1,142 @@
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        private IList<View> viewList;
+
+        private View frameView;
+
+        /// <summary>
+        /// The index of the selected View.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected int selectedIndex { get; set; }
+
+        /// <summary>
+        /// Creates a new instance of TabContent.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public TabContent()
+        {
+            selectedIndex = -1;
+            viewList = new List<View>();
+
+            frameView = new View();
+            frameView.WidthSpecification = LayoutParamPolicies.MatchParent;
+            frameView.HeightSpecification = LayoutParamPolicies.MatchParent;
+            Add(frameView);
+        }
+
+        /// <summary>
+        /// Add a View to the TabContent.
+        /// </summary>
+        /// <param name="view">A View to add</param>
+        /// <exception cref="ArgumentNullException">view is null</exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public virtual void AddView(View view)
+        {
+            if (view == null)
+            {
+                throw new ArgumentNullException(nameof(view));
+            }
+
+            viewList.Add(view);
+
+            if (selectedIndex == -1)
+            {
+                Select(0);
+            }
+
+            view.WidthSpecification = LayoutParamPolicies.MatchParent;
+            view.HeightSpecification = LayoutParamPolicies.MatchParent;
+        }
+
+        /// <summary>
+        /// Remove a View from the TabContent.
+        /// </summary>
+        /// <param name="view">A View to remove</param>
+        /// <exception cref="ArgumentNullException">view is null</exception>
+        /// <exception cref="ArgumentException">view is not in the TabContent</exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public virtual void RemoveView(View view)
+        {
+            if (view == null)
+            {
+                throw new ArgumentNullException(nameof(view));
+            }
+            if (!viewList.Contains(view))
+            {
+                throw new ArgumentException("This TabContent doesn't contain ", nameof(view));
+            }
+
+            int index = viewList.IndexOf(view);
+
+            viewList.Remove(view);
+
+            if (index == selectedIndex)
+            {
+                if (viewList.Count == 0)
+                {
+                    Select(-1);
+                }
+                else if (selectedIndex == viewList.Count)
+                {
+                    Select(selectedIndex - 1);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Select one object from the Views.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public virtual void Select(int index)
```

Comment:
I think TabContent.Select(int) does not need to be public.
`Internal` looks fine.
Why don't you add public `Select` method in TabView?

Reply:
@rabbitfor 
Is it OK if TabContent.Select(int) is changed from public to internal?
Because TabView should set the current selected index of TabContent by calling that method.